### PR TITLE
Add Set Lesson command and Update GUI 

### PR DESCRIPTION
**Pull request by seanpzk**
Set Lesson command:
- Command format: `setlesson /c CLASS_NAME /l NUM_OF_LESSONS`
- Note: When total number of lessons in a `Class` is set to be lower, it checks through ALL the `Student`'s number of lessons attended. It ensures `Student`'s number of lessons do not exceed total number of lesson in a `Class`. If it exceeds, `Student`'s number of lessons attended is reduced to the new total number of lessons in a `Class`.

GUI:
- Shows only Student's ID below Student's name

UserGuide:
- Added whitespace for better presentation
- Update format of `RemoveStudent`, `SetLesson`, `StartLesson`
- Update with google colab

### CODE REVIEW

**Review by revdrag — APPROVED** [review]
Thanks for updating the table of contents in the ug!

File: src/main/java/seedu/address/logic/commands/SetLessonCommand.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            + PREFIX_LESSONQUANTITY + " NUMBER_OF_LESSON"
+            + "\n"
+            + "Example: "
+            + COMMAND_WORD + " cs2103" + " /l 5";
+    public static final String MESSAGE_START_LESSON_SUCCESS = "Successfully set the number of lessons in %s to %d!";
+    private ClassName className;
+    private int numLessons;
+
+    /**
+     * Command to simulate starting a lesson of a Class thus increasing total lessons conducted of the Class by one.
```
Comment (DonovanJJ) [inline src/main/java/seedu/address/logic/commands/SetLessonCommand.java]: Code looks neat and clean! 
But the javadoc here seems to be for another command

File: src/main/java/seedu/address/logic/parser/SetLessonCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import seedu.address.logic.commands.SetLessonCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.module.ClassName;
+
+/**
+ * Parses input arguments and create a new SetLessonCommand.
+ */
+public class SetLessonCommandParser {
+    /**
+     * Parses the input arguments and create a SetLessonCommand based on input arguments.
```
Comment (DonovanJJ) [inline src/main/java/seedu/address/logic/parser/SetLessonCommandParser.java]: Slight grammatical error here. Should be "creates" instead of "create"

**Review by DonovanJJ — APPROVED** [review]
LGTM! Just some nitpicks here and there 👍

**Pull request merged**